oracle/price-feeder/oracle/oracle.go

```diff
@@ -121,23 +111,20 @@ func (o *Oracle) Start(ctx context.Context) error {
 			currBlockHeight := <-o.oracleClient.BlockHeightEvents
 
 			// Track how many pending goroutines are there
```

Comment:
is this comment still correct?

Reply:
Good catch, it's not needed since we switch to single threaded mode